src/utils/__tests__/test-html-converter.ts

```
@@ -0,0 +1,120 @@
+          style: 'normal',
+          children: [{
+            _type: 'span',
+            _key: nanoid(),
+            text: element.textContent || '',
+          }],
+          markDefs: [],
+        })
+        break
+        
+      case 'figure':
+        const audio = element.querySelector('audio')
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)